telco-ran/install/README.md

```diff
@@ -0,0 +1,24 @@
+# Installation artifacts
+Note: This repository is a work in progress and might be subject to structural change.
```

Comment:
It would be worth adding that this method is deprecated in favor of the ClusterInstance CR from the siteconfig operator and include a link to that [repo](https://github.com/stolostron/siteconfig).